### PR TITLE
Make instrument light selectable

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Comment by gilbertohasnofb** [comment]
@D-ECHO this seems a bit messy, why are you having textures modifications in this PR (e.g. `sc7-top.png ` or `blank.png`?) This might conflict with #97

**Comment by gilbertohasnofb** [comment]
I just realized that you made the instrument `vario/ilec-sc7` local, and so did I in that other PR. Perhaps it would be better to merge this first as I have modifications to the textures as well.

**Comment by gilbertohasnofb** [comment]
I just tested this PR and everything seems to work well, but just a couple of things:

- since now you clarified that this is not a RL feature, then are you sure we want to leave this option for lights? On my screen there is no difference of readability between having lights on or lights off, but lights off looks way better
- if you really want this feature, then I would suggest at least that the default option should be lights off and not on

**Comment by ghost** [comment]
it's now off by default plus ground shadow is added!

**Comment by gilbertohasnofb** [comment]
Great, let me try again. Then I will merge this and solve any conflicts with the other PR I pushed

**Comment by gilbertohasnofb** [comment]
@D-ECHO looking good, nearly there. But it looks to me that the shadow is pitch black, shouldn't it have a bit more of transparency like in the c172p? See:

fgfs-screen-002

**Comment by gilbertohasnofb** [comment]
Also, I would not make the canopy of the shadow half transparent, as it looks very strange from most angles (for instance in the image above, the glass wouldn't influence the shadow but rather the bottom of the plane!)

**Comment by ghost** [comment]
Last commit makes shadow a bit transparent and fixes the window issue (by making it two-sided), also should fix #94

**Comment by gilbertohasnofb** [comment]
@D-ECHO Great, taking a look

**Comment by gilbertohasnofb** [comment]
Looks good, I am merging this. It's a pity that shadows are a bit buggy in FG (perhaps someone could talk to Thorsten about it), see for instance these two images:

fgfs-screen-003

fgfs-screen-004

**Comment by ghost** [comment]
Can be deleted?
